spyder/plugins/editor/widgets/tests/test_save.py

```diff
@@ -335,3 +366,4 @@ def test_save_all(editor_bot, mocker):
 
 if __name__ == "__main__":
     pytest.main()
+    # pytest.main(['-x', osp.basename(__file__), '-v', '-rw'])
```

Comment:
Please remove this comment. Also, if you want to run a particular file with pytest, you can do it with

    pytest spyder/plugins/editor/widgets/tests/test_save.py

That has the additional advantage of running it with a clean config settings.